supabase/migrations/10_spec_ext.sql

```diff
@@ -84,6 +84,15 @@ comment on view combined_grants_ext is
 
 -- Extended view of live catalog specifications.
 create view live_specs_ext as
+with user_read_access AS (
+    select
+        distinct role_prefix,
+        max(capability) as capability
```

Comment:
nit: drop `capability` as a selected column (it's not used), as well as the `group by` (implied by distinct)